src/nvim/lua/vim.lua

```diff
@@ -38,8 +38,8 @@
 -- Returns:
 --    {errcode}, {output}
 local function _system(cmd)
-  local out = vim.api.nvim_call_function('system', { cmd })
-  local err = vim.api.nvim_get_vvar('shell_error')
+  local out = vim.api.nvim_call_function('system', { cmd }) --luacheck: ignore 113
+  local err = vim.api.nvim_get_vvar('shell_error') --luacheck: ignore 113
```

Comment:
We should have a `.luacheckrc` instead, with:
```lua
-- Global objects defined by the C code
read_globals = {
    "vim",
}
```

Reply:
See https://github.com/awesomewm/awesome/blob/3e3c34bdd194cf75cc16a87e82b11f8df3e7c5de/.luacheckrc for more ideas.

Reply:
Also please consider using `[skip appveyor]` with following commits (since linting is run on Travis).

Reply:
Okay, I have added the .luacheck rc in src/ and removed the luacheck ignore flags for vim global var in this file.

Reply:
For future commits, please use `[skip appveyor]` with the commit subject (first line).